### PR TITLE
AWS-ECS-AutoStopping-Tag-Support

### PR DESCRIPTION
**Pull request by charanya-j**
Updated the topic to include support for tags while creating an autostopping rule for ECS clusters.

### CODE REVIEW

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ The **AutoStopping Summary of Rules dashboard** displays data as a chart and t
 
 ### Start the Instances from the Dashboard
```
Comment (michaelcretzman) [inline docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md:75]:
```suggestion
### Start the instances from the dashboard
```

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ The **AutoStopping Summary of Rules dashboard** displays data as a chart and t
...
 
-You can start an instance from the Summary of Rules Page or from the details page of a selected rule.
+You can start an instance from the Summary of Rules Page or from the Details page of a selected rule.
 
 #### From the Summary of Rules Page
```
Comment (michaelcretzman) [inline docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md:79]:
```suggestion
#### From the Summary of Rules page
```

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ You can start an instance from the Summary of Rules Page or from the details pag
 
 #### From the Details Page
```
Comment (michaelcretzman) [inline docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md:91]:
```suggestion
#### From the Details page
```

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md [hunk 4]
```diff
@@ -74,11 +74,42 @@ Select the cloud resources that you want to manage using this rule. AutoStopping
...
    
      ![](./static/create-auto-stopping-rules-for-ecs-05.png)
-2. In **Select ECS Service**, select a region and a cluster to see all the services:
-	1. Select the region where your cluster is hosted from the **All** **Regions** drop-down list.
-	2. Select your cluster from the **All Clusters** drop-down list.
-	3. Select the ECS Service for which you want to enable AutoStopping Rule and click **Add Selected**.![](./static/create-auto-stopping-rules-for-ecs-06.png)
-3. In **Desired Task Count**, specify the desired task count for the selected ECS service. This is the number of tasks that Harness will instantiate when your service is up and running. For more information, see [Amazon ECS service quotas](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-quotas.html).
+2. In **Select ECS Service**, select one of the following options:
+   * ECS Service Name
+   * ECS Service Tags
```
Comment (michaelcretzman) [inline docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:79]:
```suggestion
   * ECS Service Name.
   * ECS Service Tags.
```

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md [hunk 5]
```diff
@@ -74,11 +74,42 @@ Select the cloud resources that you want to manage using this rule. AutoStopping
...
+   
+    
+
+  If you have chosen **ECS Service Name**, select a region and a cluster to see all the services:
+  1. Select the region where your cluster is hosted from the **Select Region** dropdown list.
+  2. Select your cluster from the **Select Cluster** dropdown list.
+  3. Select the ECS Service for which you want to enable AutoStopping Rule. 
+  4. Click **Add Selected**.
```
Comment (michaelcretzman) [inline docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:87]:
```suggestion
  If you chose **ECS Service Name**, select a region and a cluster to see all of their services:
  1. Select the region where your cluster is hosted from the **Select Region** dropdown list.
  2. Select your cluster from the **Select Cluster** list.
  3. Select the ECS Service where you want to enable the AutoStopping Rule. 
  4. Click **Add Selected**.
```

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md [hunk 6]
```diff
@@ -74,11 +74,42 @@ Select the cloud resources that you want to manage using this rule. AutoStopping
...
+   
+   ![](./static/aws-ecs-tags.png)
+   
+If you have chosen **ECS Service Tags**, select the region, cluster, and the tag associated with the service. Once you create a rule using this option, it is applied automatically to the most recently created ECS service to which the tag is added.
```
Comment (michaelcretzman) [inline docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:91]:
```suggestion
If you have chose **ECS Service Tags**, select the region, cluster, and the tag associated with the service. Once you create a rule using this option, the rule is applied automatically to the most recently created ECS service using the tag. 
```

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md [hunk 7]
```diff
@@ -74,11 +74,42 @@ Select the cloud resources that you want to manage using this rule. AutoStopping
...
+  4. Click **Add Selected**.
+   
+   ![](./static/aws-ecs-tags.png)
+   
+If you have chosen **ECS Service Tags**, select the region, cluster, and the tag associated with the service. Once you create a rule using this option, it is applied automatically to the most recently created ECS service to which the tag is added. 
+1. Select the region where your cluster is hosted from the **Select Region** dropdown list.
+2. Select your cluster from the **Select Cluster** dropdown list.
+3. Select the tag key and the tag value from the dropdown lists. If you want to learn how to add tags to your service, go to [Tagging your Amazon ECS Resources](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-using-tags.html).
+   
+     You can view the list of ECS services to which this tag is added.
```
Comment (michaelcretzman) [inline docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:96]:
```suggestion
     You can view the list of ECS services using this tag. 
```

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md [hunk 8]
```diff
@@ -74,11 +74,42 @@ Select the cloud resources that you want to manage using this rule. AutoStopping
...
+
+
+
+![](./static/aws-ecs-use-tags.png)
+
+   
+### **Specify the desired task count**
+In **Desired Task Count**, specify the desired task count for the selected ECS service. This is the number of tasks that Harness will instantiate when your service is up and running. For more information, see [Amazon ECS service quotas](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-quotas.html).
 
 ### (Optional) Set Up Advanced Configuration
```
Comment (michaelcretzman) [inline docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:114]:
```suggestion
### (Optional) Set up advanced configuration
```

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md (hunk 1 above)
Comment (michaelcretzman) [reply docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md:75]: Sentence case for titles. You should probably do a pass on the whole topic for this.

**Review by michaelcretzman — DISMISSED** [review]
Approved with the assumption that you will consider my suggestions.

**Review by sujeesh-harness — APPROVED** [review]

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md (hunk 1 above)
Comment (charanya-j) [reply docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/autostopping-dashboard.md:75]: Done

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md (hunk 4 above)
Comment (charanya-j) [reply docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:79]: Done

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md (hunk 8 above)
Comment (charanya-j) [reply docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:114]: Done

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md (hunk 7 above)
Comment (charanya-j) [reply docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:96]: Done

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md (hunk 6 above)
Comment (charanya-j) [reply docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:91]: Done

File: docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md (hunk 5 above)
Comment (charanya-j) [reply docs/cloud-cost-management/2-use-cloud-cost-management/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/3-create-auto-stopping-rules/create-auto-stopping-rules-for-ecs.md:87]: Done

**Pull request merged**